live-site/components/event-schedule-section/event-item/EventItem.styles.ts

```diff
@@ -156,20 +151,51 @@ const StyledPinpoint = styled.img`
   } ;
 `;
 
-const StyledArrow = styled.img`
+const StyledArrow = styled.img<StyledArrowProps>`
```

Comment:
Here's a fix for the arrow issue: 

It works better with the rotate so we can use that instead. The only thing is when its flipped we want to apply padding to thebottom if we want it to go down more (since its flipped) and same for the padding-left to make it more to the left if that makes sense - please ask me if u have any questions!

```
const StyledArrow = styled.img<StyledArrowProps>`
  cursor: pointer;
  padding-top: ${(StyledArrowProps): string =>
    StyledArrowProps.isOpen ? `0` : `3em`};
  padding-right: ${(StyledArrowProps): string =>
    StyledArrowProps.isOpen ? `0` : `3em`};
  padding-bottom: ${(StyledArrowProps): string =>
    StyledArrowProps.isOpen ? `3em` : `0`};
  padding-left: ${(StyledArrowProps): string =>
    StyledArrowProps.isOpen ? `3em` : `0`};
  float: right;
  transform: ${(StyledArrowProps): string =>
    StyledArrowProps.isOpen ? `rotate(180deg)` : `rotate(0deg)`};
`;
```